scripts/fast-reboot

```diff
@@ -817,8 +817,8 @@ trap '' EXIT HUP INT QUIT TERM KILL ABRT ALRM
 # this is a non-blocking call, and the process will die in 300s
 debug "Starting lag_keepalive to send LACPDUs ..."
 timeout 300 python ${LAG_KEEPALIVE_SCRIPT} &
-# give the lag_keepalive script a chance to get ready (30s) and collect one lacpdu before going down (30s)
-sleep 60
+# give the lag_keepalive script a chance to send some LACPDUs
+sleep 10
```

Comment:
This approach still uses a wait approach, and the assumption is that the LACPDUs will be sent.

In mostly all platforms, sending these LACPDUs has become a prerequisite. So, may be we should do a quick capture here to ensure that LACPDUs are being sent out via (all?) LAGs. If not, abort warm-reboot.

Reply:
So doing a capture here might make things a bit messy. Instead, I added logic in the Python script to fork into the background after collecting information about the LAGs. That way, if there's an issue at this stage, then it'll get caught.